client/devdocs/design/search-collection.jsx

```diff
@@ -64,7 +65,7 @@ const Collection = ( { children, filter, section = 'design', component } ) => {
 				unique={ !! component }
 				url={ exampleLink }
 			>
-				{ example }
+				{ component ? <PropsViewer component={ component } example={ example } /> : example }
```

Comment:
This is intentional, such that the `PropsViewer` could change the props to the the `example` component. Maybe I'm doing it wrong? I thought about making it a child, but that felt strange since the props are a property of the component and if the component changes in any way, I'd like the props table to rerender. The table, as it stands in this PR, isn't very smart, but it can be quite smart (and show you the props as the example is rendering them).